catch/unit/module/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+set(TEST_SRC
+    hip_module_common.cc
+    hipModuleLoad.cc 
+    hipModuleLoadData.cc 
+    hipModuleLoadDataEx.cc 
+    hipModuleUnload.cc 
+)
+
+configure_file(not_a_module.txt not_a_module.txt)
+
+add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/empty_module.code
+                   COMMAND ${CMAKE_CXX_COMPILER} --genco --std=c++17 ${CMAKE_CURRENT_SOURCE_DIR}/empty_module.cc -o empty_module.code
+                   DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/empty_module.cc)   
```

Comment:
genco does not work properly on windows yet. you will need to exclude this test building on windows. additionally you are missing specifying the offload arch & output path for building the code object. see unit/deviceLib/CMakeLists.txt for an example of how to do it for all of the above.

Reply:
Requested changes have been implemented, please proceed.